Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM nfcore/base
-MAINTAINER Phil Ewels <phil.ewels@scilifelab.se>
-LABEL authors="phil.ewels@scilifelab.se" \
-    description="Container image containing all requirements for the nf-core/methylseq pipeline"
+FROM nfcore/base:1.7
+LABEL authors="Phil Ewels" \
```

Comment:
We don't email anymore in the Dockerfile?

Reply:
No we don't - emails change too often ;)

Reply:
emails or person?

Reply:
Main reason I think is that the `nf-core create` tool takes the author name and not the author email. We _could_ update it to handle both fields, but it would only be used here and may put some people off. I figure it's pretty easy to find people to get in touch with regarding the pipeline and container, so it's not really required.